pychunkedgraph/backend/chunkedgraph.py

```diff
@@ -3054,8 +3055,14 @@ def add_edges(self, user_id: str, atomic_edges: Sequence[np.uint64],
                                    operation_id=operation_id,
                                    slow_retry=False):
                     if remesh_preview:
-                        meshgen.mesh_lvl2_previews(self, list(
-                            lvl2_node_mapping.keys()))
+                        if use_mesh_celery_worker:
+                            serialized_cg_info = self.get_serialized_info()
+                            del serialized_cg_info["credentials"]
+
+                            worker.mesh_lvl2_preview(serialized_cg_info, )
+                        else:
+                            meshgen.mesh_lvl2_previews(self, list(
+                                lvl2_node_mapping.keys()))
```

Comment:
There is an additional location where preview remeshing is triggered down in remove_edges.